.github/workflows/test.yml

```diff
@@ -36,6 +47,16 @@ jobs:
         pip install -r requirements.txt
         pip install -r dev-requirements.txt
         pip install -e .
+
+    - name: Set Git safe directory
+      run: git config --global --add safe.directory /__w/ckanext-short-urls/ckanext-short-urls
+
+    - name: Checkout tag 1.0.0
```

Comment:
This feels like a sub-optimal engineering choice here for me... it means that if we ever make a change to the v2.9.0-compatible extension that we need to manually update this workflow... and that doesn't feel nice.

Perhaps the tag should include the compatibility? e.g instead of `v1.0.0` let's use `v1.0.0-ckan2.9` and then we check out the latest tag that includes `ckan2.9`?

Or... maybe we consider all `v1.*.*` to be compatible with v2.9, all `v2.*.*` to be compatible with v2.10... and then what do we do with v2.11? It doesn't _look_ like there have been any breaking changes between v2.10 and v2.11 that would impact this extension.

Whatever we do, this information needs to be in the README